### PR TITLE
Visual improvements on error pages.

### PR DESCRIPTION
**Pull request by glauber-sampaio**
Visual improvements to make it feel more minimalistic.
screen shot 2016-04-16 at 2 18 21 pm

### CODE REVIEW

File: css/navbarTabs.css [hunk 1]
```diff
@@ -48,7 +48,7 @@ body.notMac .tab-group {
 	min-width: 125px;
 	transition: .15s min-width, 0.2s transform;
 	padding: 0 1em;
-	font-size: 0.8em;
+	font-size: 0.4em;
```
Comment (PalmerAL) [inline css/navbarTabs.css]: What's the reason for this change?

File: pages/pagebase.css [hunk 2]
```diff
@@ -19,15 +19,18 @@
 	h1 {
 		font-size: 2.75em;
 		font-weight: normal;
-		margin-bottom: 0.25em;
+		margin-bottom: 0.45em;
 		margin-top: 0;
 	}
 	h2 {
```
Comment (PalmerAL) [inline pages/pagebase.css]: This text is kind of hard to read. Could this have less opacity, or be a little bit larger?

**Comment by PalmerAL** [comment]
This looks really nice! Would it be possible to not use ```<br>``` tags to split the lines? If you shrink the window down small enough, the text gets split in the middle of a line:

screen shot 2016-04-16 at 1 11 05 pm

**Comment by glauber-sampaio** [comment]
Got it. I'm addressing your feedback on my local branch and doing a lot of new improvements on design.

**Comment by PalmerAL** [comment]
I'm going to close this PR since it hasn't been active for a while. Feel free to re-open it if you want to continue working on it!
